uint64 timestamp
float64 latency_secs
uint64 send_time_ns
uint64 recv_time_ns
uint64 msg_size
bool is_inter_process
